client/selfrepair/tests/test_normandy_driver.js

```diff
@@ -36,21 +39,106 @@ describe('Normandy Driver', () => {
     });
   });
 
+  describe('saveHeartbeatFlow', () => {
+    const driver = new NormandyDriver();
+
+    afterEach(() => {
+      fetchMock.restore();
```

Comment:
Should we do this in `afterEach` or `beforeEach`? I don't really care either way, but we should probably be consistent. We did it in `beforeEach` in [another test](https://github.com/brittanystoroz/normandy/blob/149c63c7afb47268c723f0fbbf0bffa2699c5814/client/control/tests/actions/test_controlActions.js#L13-L20).